src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -55,6 +56,12 @@ def opendir(self, dirpath):
         try:
             d = self.cfs.opendir(dirpath)
             yield d
+        except cephfs.OSError:
+            dirpath = dirpath.decode().replace("/.snap", "")
+            raise DashboardException(code='invalid_dir_permission',
+                                     msg="Unable to open directory: {} due to invalid permissions.\
+                                     Try adding the +x permission to the directory".format(dirpath),
+                                     component='cephfs')
```

Comment:
If we want to enforce ourselves/future-Dashboard-devels to follow the "title-what happened-how to resolve" error reporting pattern, perhaps we need to extend the `DashboardException` or derive a new exception (`NotifyError()`?) with a couple of new fields (`msg` could be the `what happened`): `DashboardException(..., title=None, resolution=None)`=